src/components/ui/bento-grid.tsx

```
@@ -0,0 +1,54 @@
+      className={cn(
+        "grid md:auto-rows-[18rem] grid-cols-1 md:grid-cols-3 gap-4 max-w-7xl mx-auto ",
+        className
+      )}
+    >
+      {children}
+    </div>
+  );
+};
+
+export const BentoGridItem = ({
+  className,
```

ESLint

[error] Replace `··` with `····`

(prettier/prettier)